plenum/server/replica.py

```diff
@@ -1749,14 +1808,17 @@ def order_3pc_key(self, key):
             )
 
         self.addToOrdered(*key)
+        valid_reqs, invalid_reqs = self._apply_bitmask_to_list(pp.reqIdr,
```

Comment:
Why do we call it here?

Reply:
I see, if use this info in logging further, then probably pass valid_reqs and invalid_reqs to Ordered msg to avoid the same calculation in node.py?